salt/cloud/clouds/digital_ocean.py

```diff
@@ -845,7 +845,12 @@ def destroy_dns_records(fqdn):
     '''
     domain = '.'.join(fqdn.split('.')[-2:])
     hostname = '.'.join(fqdn.split('.')[:-2])
-    response = query(method='domains', droplet_id=domain, command='records')
+    # TODO: remove this when the todo on 754 is available
+    try:
+        response = query(method='domains', droplet_id=domain, command='records')
+    except SaltCloudSystemExit:
+        log.debug('Failed to find domains.')
```

Comment:
I am ok with either, but as this is run every time and can't be turned off I had it as debug so it wouldn't show up all the time.